Tools/Makefile

```diff
@@ -63,6 +63,9 @@ clean:
 	@$(build_target_for_each_module)
 
 installsrc:
-	@true
+ifeq ($(FORCE_TOOL_INSTALL),YES)
+	@$(build_target_for_each_module)
+	ditto TestRunnerShared "$(SRCROOT)/Tools/TestRunnerShared"
+endif
```

Comment:
Removing `@true` will mean that make starts printing

    make: Nothing to be done for `installsrc'.

but I think that's fine. It still exits with `0`, so it shouldn't break our scripts.

Reply:
Oh, I can add an else. Didn't realize wha the consequence was, thought it was syntax only.